provider/aws/cloudformation.go

```diff
@@ -345,6 +366,15 @@ func buildStack(stackDetails *cloudformation.Stack) *common.Stack {
 	} else {
 		stack.LastUpdateTime = aws.TimeValue(stackDetails.CreationTime)
 	}
+	// parse Region and Account out of ARN
+	arnChunks := strings.Split(stack.ID, ":")
+	if len(arnChunks) > 3 {
+		stack.Region = arnChunks[3]
+	}
+	if len(arnChunks) > 4 {
+		stack.AccountID = arnChunks[4]
+	}
```

Comment:
doesn't seem these are used...is this old?

Reply:
This is the place where AccountID and Region were populated, but not otherwise referenced. I removed this too.